homeassistant/components/egps/config_flow.py

```diff
@@ -0,0 +1,56 @@
+"""ConfigFlow for EGPS devices."""
+
+from typing import Any
+
+from pyegps import get_device, search_for_devices
+from pyegps.exceptions import MissingLibrary, UsbError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+
+from .const import CONF_DEVICE_API_ID, DOMAIN, LOGGER
+
+
+class EGPSConfigFlow(ConfigFlow, domain=DOMAIN):
```

Comment:
So this uses USB. Does USB discovery work here?

Reply:
Unfortunately not that easy, if I interpreted the code correctly the USB discovery of HA only works with usb to serial converters. This USB device here reports itself as a HID device.